bin/librato-storm-kafka

```
@@ -236,13 +236,18 @@ def do_thr()
     break if work.nil?
 
     begin
-      work = monitor_partition(work)
+      result = monitor_partition(work)
     rescue Errno::ETIMEDOUT, Errno::EHOSTUNREACH => e
```

Does it make sense to catch all exceptions here instead?